src/Binding/JSModules.cpp

```diff
@@ -75,7 +75,8 @@ JSModule::JSModule(JSContext *cx,
                    const char *name)
     : m_AbsoluteDir(nullptr), m_FilePath(nullptr), m_Name(strdup(name)),
       m_ModuleType(JSModule::kModuleType_None), m_Cached(false),
-      m_Exports(nullptr), m_Parent(parent), m_Modules(modules), m_Cx(cx)
+      m_Exports(nullptr), m_Parent(parent), m_Modules(modules), m_Cx(cx),
+      m_dlmodule(NULL)
```

Comment:
We are now initializing values during the declaration of the variable (also we use nullptr instead of NULL) (see my comment in JSModule.h)